android/runtime/common/src/java/org/appcelerator/kroll/util/KrollAssetHelper.java

```diff
@@ -300,18 +300,6 @@ private static String normalizeDirectoryPath(String path)
 		return path;
 	}
 
-	private static String normalizeAssetCryptPath(String path)
-	{
-		final String ROOT_PATH_NAME = "Resources/";
-
-		if (path != null) {
-			if (path.startsWith(ROOT_PATH_NAME)) {
-				path = path.substring(ROOT_PATH_NAME.length());
-			}
-		}
-		return path;
-	}
-
```

Comment:
Theoretically, removing the `normalizeAssetCryptPath()` method to strip off the leading "Resources/" from the path would break backward compatibility with a hybrid module's `AssetCryptImpl` generated code.

This means our "ti.playservices" module's `makeGooglePlayServicesAvailable()` method might not be accessible anymore

To be fair, I don't like that our old code was stripping off the "Resources/" directory from the path to begin with. It's a bad assumption on our part. But we kept this part of the code alive for backward compatibility reasons. If we can prove that this is no longer needed anymore, then yes, let's remove this method.

Reply:
Existing hybrid modules work fine, I tested `ti.playservices`. I also removed the dependency of `titanium_prep` when compiling hybrid modules https://github.com/appcelerator/titanium_mobile/pull/11045/commits/1d47632d2ef06057e58bd6a1a27a2e5a1385b09a

Reply:
Okay cool. Thanks for testing it out. Then yes, we should get rid of this method.